chain/client/src/client.rs

```diff
@@ -176,6 +177,8 @@ pub struct Client {
     /// Cached precomputed set of TIER1 accounts.
     /// See send_network_chain_info().
     tier1_accounts_cache: Option<(EpochId, Arc<AccountKeys>)>,
+    /// Takes care of performing resharding on the flat storage.
+    pub flat_storage_resharder: FlatStorageResharder,
```

Comment:
This may be moved inside the resharder manager @shreyan-gupta

Reply:
Yeah, agree if it's simple enough